CODEOWNERS

```diff
@@ -2,7 +2,47 @@
 /erroranalysis @gaugup @imatiach-msft
 
 # Responsible AI widgets package
-/raiwidgets @imatiach-msft @xuke444 @zhb000 @vinuthakaranth @RubyZ10 @tongyu-microsoft @romanlutz
+/raiwidgets @imatiach-msft @xuke444 @romanlutz
```

Comment:
Can we add more folks to raiwidgets?  Just seems like 3 is very few for the largest python package we have.

Reply:
I don't particularly care about the number tbh. The people who are listed should know what the code does, though. I can only speak for myself, of course, and feel like this PR represents which parts I know enough about to review PRs. In some ways it's better to have tighter control while encouraging more reviews per PR. For example, even if I'm not required for the responsibleai package it's probably still useful if I provide feedback. At the same time, that alone shouldn't be sufficient to merge a PR. So in summary, to me this makes sense.

Regarding size: Is raiwidgets really that large? I feel like a good chunk of it is in other packages (responsible, rai-core-flask, etc.) so it's a fairly thin layer of things. It is the one that's most exposed to users, of course.

Wdyt @imatiach-msft?

Reply:
Hmm maybe responsibleai package is actually the largest in terms of the number of lines of python code.  However, raiwidgets in some sense also contains all of the typescript code, so it is still the largest by total lines of python + typescript code, from that viewpoint.  Regardless it is the main package that depends on all of the others, so it's the most important one for us, and it often has a lot of code changes in PRs.

Reply:
I think you mix folder with package.
This code owner is based on file structure not pipy package.
raiwidget directory has 0 typescript
it is mainly handle how we plug flask to typescript.
That's why I only put limited amount of people there.
all the typescript ui code splitted into different folders have different approvers.